### ISSUE TITLE
Check 16 - Exclusions list for content databases

### ISSUE THREAD

**Issue by JvanderMeer-GitHub** [ykuijs/SPPeriodicChecks#4]
It would be great to exclude certain databases.  I made some changes in the check itself.

```Powershell
$siteThreshold = 5 #Minimal amount of free spaces.
$dbIgnore = @(".*IGNORE-DATABASENAME-B.*",".*IGNORE-DATABASENAME-B.*") #Databases to ignore by check
$spcdbs = @()

$errorCount = 0
$errorDB    = ""
$spcdbs = Get-SPContentDatabase
$cdbs = $spcdbs -notmatch ($dbIgnore -join "|")

foreach ($cdb in $cdbs)
{
    $cdbSize = $cdb.MaximumSiteCount - $cdb.CurrentSiteCount
    if (($cdbSize -lt $siteThreshold) -and ($cdbSize -ne 0) -and ($cdbSize -gt 0))
    {
        $errorCount++
        if ($errorDB -ne "")
        {
            $errorDB += ", "
        }
        $errorDB += "$($cdb.Name) (Free: $cdbSize)"
    }
}
```

**Comment by ykuijs** [comment]
Great idea! Will add this to the script.

**Comment by ykuijs** [comment]
Added in next version

**Comment by ykuijs** [comment]
Added in next version.

Same logic implemented for new check 19.

**Comment by ykuijs** [comment]
Fixed via PR #12
